source/sass/weekly-product/weekly-product.scss

```
@@ -12,7 +12,7 @@
     bottom: 0;
     left: 0;
     background: 0 0 url("../../img/bg-zigzag-line.svg") repeat-x;
```

Здесь сделай вот так:

```suggestion
    background: center center url("../../img/bg-zigzag-line.svg") repeat-x;
```

Полоска по-прежнему будет обрезаться, но теперь с двух сторон, что будет выглядеть аккуратнее.